src/main/java/core/basesyntax/Model/Cat.java

```diff
@@ -1,4 +1,4 @@
-package core.basesyntax.Model;
+package core.basesyntax.model;
```

Comment:
Same about package name, please pay more attention to how your actual folder is named and what is provided in `package` statement.